exa/core/box.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2015-2020, Exa Analytics Development Team
+# Distributed under the terms of the Apache License 2.0
+"""
+Data container
+####################################
+An in-memory related data container.
+"""
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import networkx as nx
+from traitlets import Unicode, Dict
+
+import exa
+from exa import Base
+from exa.core.data import Data
+
+class Box(Base):
```

Comment:
Is my understanding correct that this is the "side-by-side" `Container 2.0` development?

Reply:
Yeah, you know how naming things goes. `Box` was the best I could do.